src/tests/functions.at

```diff
@@ -392,6 +392,7 @@ m4_define([NFT_LIST_RULES_NORMALIZE], [dnl
         -e '/type.*hook.*priority.*policy.*/d'dnl
         dnl tranform ct state { established,related } to ct state established,related
         -e '/ct \(state\|status\)/{s/\(ct \(state\|status\)\) {/\1/g; s/ }//; s/\(@<:@a-z@:>@*\), /\1,/g;}' dnl
+        -e 's/reject with icmp[[x6]]\? type port-unreachable/reject/' dnl
```

Comment:
ICMPv6 syntax is: 'reject with icmpv6 ...'. So the match part should read: 'reject with icmp\(x\|v6\)\? type port-unreachable'.

Reply:
Thanks for catching that. I'll push a follow up.

Reply:
I directly pushed commit 3a3b4676ccb7b40cf304b773456dec2662783425.